nginx.conf.d/ssl.conf

```diff
@@ -0,0 +1,5 @@
+# Shared SSL settings for all hosts.
```

Comment:
There's not tons to share right now, but soon I'm going to go through and test mozilla's recommendations the staging server.